lightkurve/correctors.py

```diff
@@ -487,3 +490,95 @@ def breakpoints(self, campaign):
         independently. However, this is not implemented yet in this version.
         """
         raise NotImplementedError()
+
+class PLDCorrector(object):
+    """Implements the Pixel Level Decorrelation (PLD) method to de-trend
+    Target Pixel Files.
+    """
+
+    def __init__(self, tpf):
+        self.tpf = tpf
+        self.fpix = np.nan_to_num(tpf.flux)
+        self.flux_err = np.nan_to_num(tpf.flux_err)
+        self.time = tpf.time
+
+    def correct(self, transit_mask=[], aperture_mask=None):
+        """Returns a systematics-corrected LightCurve.
+
+        Note that it is assumed that time and flux do not contain NaNs.
+
+        Parameters
+        ----------
+        time : array-like
+            Time measurements
+        flux : array-like
+            Data flux for every time point
+
+        Returns
+        -------
+        corrected_lightcurve : LightCurve object
+            Returns a corrected lightcurve object.
+        """
+
+        if aperture_mask is None:
+            aperture_mask = self.tpf.pipeline_mask
```

Comment:
I recommend replacing this with `aperture_mask = tpf._parse_aperture_mask(aperture_mask)` so that it supports exactly the same syntax as elsewhere (e.g. this will add support for `aperture_mask='threshold'` etc.)

You can still define your own default by e.g. setting `aperture_mask='threshold'` in the function signature on line 505.